src/integrations/sentry_integration_qt.cpp

```diff
@@ -0,0 +1,59 @@
+#include "sentry_integration_qt.h"
+#include "sentry_boot.h"
+
+#include <QtCore/qglobal.h>
+#include <QtCore/qstring.h>
+
+static QtMessageHandler originalMessageHandler = nullptr;
+
+static const char *
+logLevelForMessageType(QtMsgType msgType)
+{
+    switch (msgType) {
+    case QtDebugMsg:
+        return "debug";
+    case QtWarningMsg:
+        return "warning";
+    case QtCriticalMsg:
+        return "error";
+    case QtFatalMsg:
+        return "fatal";
+    case QtInfoMsg:
+        Q_FALLTHROUGH();
+    default:
+        return "info";
+    }
+}
+
+static void
+sentry_qt_messsage_handler(
+    QtMsgType type, const QMessageLogContext &context, const QString &message)
+{
+    sentry_value_t crumb
```

Comment:
I wonder if we can have a better value for `type` here.
Perhaps even `NULL` is better, the UI adjusts to deal with the data it has and it'll have category and level already.

```
type (optional)
The type of breadcrumb. By default, all breadcrumbs are recorded as default, which makes them appear as a Debug entry, but Sentry provides other types that influence how the breadcrumbs are rendered. For more information, see the description of recognized breadcrumb types.

By default, all breadcrumbs are recorded as default, which makes them appear as a Debug entry, but Sentry provides other types that influence how the breadcrumbs are rendered. For more information, see the description of recognized breadcrumb types.
```

There are some docs here: https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types